src/abstractdataframe/show.jl

```diff
@@ -543,32 +549,70 @@ end
 #' df = DataFrame(A = 1:3, B = ["x", "y", "z"])
 #' showall(df, true)
 function Base.showall(df::AbstractDataFrame,
-                      splitchunks::Bool = false) # -> Void
-    showall(STDOUT, df, splitchunks)
+                      allcols::Bool = true) # -> Void
+    showall(STDOUT, df, allcols)
     return
 end
 
+#' @exported
 #' @description
 #'
 #' Render a summary of the column names, column types and column missingness
 #' count.
 #'
 #' @param io::IO The `io` to be rendered to.
 #' @param df::AbstractDataFrame An AbstractDataFrame.
+#' @param all::Bool If `false` (default), only a subset of columns
+#'        fitting on the screen is printed.
+#' @param values::Bool If `true` (default), the first and the last value of
+#'        each column are printed.
 #'
 #' @returns o::Void A `nothing` value.
 #'
 #' @examples
 #'
 #' df = DataFrame(A = 1:3, B = ["x", "y", "z"])
-#' showcols(df, true)
-function showcols(io::IO, df::AbstractDataFrame) # -> Void
+#' showcols(df)
+function showcols(io::IO, df::AbstractDataFrame, all::Bool = false,
+                  values::Bool = true) # -> Void
     println(io, summary(df))
     metadata = DataFrame(Name = _names(df),
                          Eltype = eltypes(df),
                          Missing = colmissing(df))
-    showall(io, metadata, true, Symbol("Col #"), false)
+    nrows, ncols = size(df)
+    if values && nrows > 0
+        # type of Values column is now String; it might need to be changed
```

Comment:
I don't think this comment is needed: tests will (or should) catch this and people will figure out what needs to be changed anyway.

Reply:
removed